src/components/bio.js

```diff
@@ -62,6 +62,14 @@ const Bio = () => {
         >
           Saffron
         </a>
+        or my newest app called{" "}
+        <a
+          href="https://dogehouse.tv/"
```

Comment:
This just leads to a youtube video. It would be better for it to go to the [Github repo](https://github.com/benawad/dogehouse).